tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -250,6 +251,17 @@ Verify "Enabled" Keeps Being Available After One Of The ISV Operators If Uninsta
    Uninstall Operator And Check Enabled Page Is Rendering  operator_name=${openvino_operator_name}  operator_appname=${openvino_appname}
    [Teardown]    Check And Uninstall Operator In Openshift    ${openvino_operator_name}   ${openvino_appname}
 
+Verify external links in Quick Starts are not broken
+        [Tags]    Tier1
+        ...       ODS-1305
+        [Documentation]    Verify external links in Quick Starts are not broken
+        Click Link                          Resources
+        Wait Until Resource Page Is Loaded
+        ${qucickStartElements}=     Wait for QuickStart to Load
+        # Select Quick Start Checkbox
+        Select Checkbox                         xpath=//*[@id="quickstart--check-box"]
```

Comment:
why? you already extracted the quick starts at the previous step. 

Try to avoid using python comments to explain. The code should be self-explanatory as much as possible